src/algebra/lie/submodule.lean

```diff
@@ -183,31 +183,48 @@ end
 
 namespace lie_subalgebra
 
-variables {L}
+variables {L} (K : lie_subalgebra R L)
+
+/-- Given a Lie subalgebra `K ⊆ L`, a Lie module `M` of `L`, and a Lie submodule `N ⊆ M` for the
+action of `L`, we may also regard `N` as a Lie submodule for the action of `K`. -/
+def restrict_lie_submodule (N : lie_submodule R L M) :
```

Comment:
This is true more generally for a morphism `K → L` or lie algebras, right? So maybe it makes sense to do that case first.

Reply:
Agreed. In fact I'm just dropping this proposed `def` from the PR since it is unrelated and I now think is best added separately (e.g., when it is needed).